### ISSUE TITLE
setValue() on Reactive Form with step 

### ISSUE THREAD

**Issue by MickL** [angular-slider/ng5-slider#86]
I am using the slider with Reactive Forms and steps. When setting the value manually the slider shows the correct, rounded, value but the FormControl has the wrong value in it.

Example:
```
options: Options = {
    floor: 10,
    ceil: 1000,
    step: 10,
 }
```

```
myFormControl.setValue(69);
```

-> Slider shows 70 (correct)
-> FormControl has value 69 (wrong)

**Comment by piotrdz** [comment]
Ah, it looks like the reactive forms counterpart of #69. I will fix it in the next release.

**Comment by piotrdz** [comment]
This is fixed by recent changes re-writing a lot of value update code. It will be available in next release, coming in next day or two.

**Comment by piotrdz** [comment]
Fix released with v1.2.0.
